include/core/Basis.hpp

```diff
@@ -308,7 +308,8 @@ class Basis {
 	Basis();
 
 	const Vector3 &operator[](int axis) const {
-		return get_axis(axis);
+        static const Vector3 val = get_axis(axis);
```

Comment:
This one's not okay. You can't reuse a single static variable across all instances of this class.

Think about what would happen with this if an application calls this from multiple threads simultaneously.

FYI, I already fixed this one in https://github.com/GodotNativeTools/godot-cpp/pull/264. Simply we should not return by reference but by value.

The rest of this PR seems fine. Thanks for fixing it.

Reply:
Thanks, rolled it back.